plugins/catalog-backend/src/ingestion/processors/AwsOrganizationProcessor.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ComponentEntityV1alpha1,
+  Entity,
+  LocationSpec,
+} from '@backstage/catalog-model';
+import AWS, { Organizations } from 'aws-sdk';
+import { Account, ListAccountsResponse } from 'aws-sdk/clients/organizations';
+
+import * as results from './results';
+import { CatalogProcessor, CatalogProcessorEmit } from './types';
+
+const AWS_ORGANIZATION_REGION = 'us-east-1';
+const LOCATION_TYPE = 'aws-organization';
+
+export class AwsOrganizationProcessor implements CatalogProcessor {
+  organizations: Organizations;
+  constructor() {
+    this.organizations = new AWS.Organizations({
```

Comment:
how does auth happen here?

Reply:
https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html

The SDK will pick up a number of credential configurations automatically. 

If custom authentication is needed, it can be achieved by configuring the global AWS.credentials object in the individual backstage app - though I do think it would be cleaner to support injecting a credentials object into this processor.

Reply:
Alright. This is good info and if it's a "limitation" or expectation of sorts that auth is set up in a certain way, I think a jsdoc comment on this class would be great (well it would either way) explaining what it does and what is necessary to make it tick.